Tools/pds_compress

```diff
@@ -367,6 +367,37 @@ def formatc(f_out, pds):
             buf = ""
     f_out.write(textwrap.dedent(tmpl_c) % out);
 
+tmpl_rust = """\
+/// PDS (platform data set) is SiLab's solution to configuring the wifi chip.
+/// Options such as GPIO, PA power, number of antennae are configured with PDS.
+/// The data set here is "compiled" using a script (pds_compressy) that
+/// can be found in SiliconLabs github repository:
+/// https://github.com/SiliconLabs/wfx-fullMAC-tools/Tools/pds_compress
+/// 
+/// The output is meant to be redirected into a .rs file in the same crate as your
+/// hal driver, and pulled in with the appropriate mod/use statements.
```

Comment:
I think that Rust output should contains a license. It should also warn the user that this file is automatically generated.

In add, I think it should be more or less identical to the C template. So, if you update the Rust template, I think you should also update the C template.

Besides that, the URL you mention return an error 404. I suggest this one:
     https://github.com/SiliconLabs/wfx-linux-tools/blob/master/pds_compress
(`pds_compress` is maintained in `wfx-linux-tools` repository. `wfx-fullMAC-tools` is a downstream.)

Reply:
thanks for the feedback.

I've added the license, and I fixed the URL. sorry about that, it was a blob reference before and I didn't check it after I removed the tag.

For the templates, the "///" is specific to rust as a documentation output trigger, and it looks like you've also tried to make the C comment trigger a doc output. Didn't want to just copy the "///" stuff over to the C header since the meta-use is a little different....